From 142aca6c65cf3cb61819906d23cff339acd961f7 Mon Sep 17 00:00:00 2001
From: Hyun Kwon <hyun.kwon@xilinx.com>
Date: Wed, 9 Nov 2016 10:37:03 -0800
Subject: [PATCH 1046/1566] drm: xilinx: fb: Don't check the fb format

commit  54d994cf7e2a1a08e4d2cdc572644dd277075629 from
https://github.com/Xilinx/linux-xlnx.git

The format checking is done by core already. No need to do in the driver.

Signed-off-by: Hyun Kwon <hyun.kwon@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/gpu/drm/xilinx/xilinx_drm_fb.c |    6 ------
 1 files changed, 0 insertions(+), 6 deletions(-)

diff --git a/drivers/gpu/drm/xilinx/xilinx_drm_fb.c b/drivers/gpu/drm/xilinx/xilinx_drm_fb.c
index b6ff889..4c77142 100644
--- a/drivers/gpu/drm/xilinx/xilinx_drm_fb.c
+++ b/drivers/gpu/drm/xilinx/xilinx_drm_fb.c
@@ -479,12 +479,6 @@ xilinx_drm_fb_create(struct drm_device *drm, struct drm_file *file_priv,
 	int ret;
 	int i;
 
-	if (!xilinx_drm_check_format(drm, mode_cmd->pixel_format)) {
-		DRM_ERROR("unsupported pixel format %08x\n",
-			  mode_cmd->pixel_format);
-		return ERR_PTR(-EINVAL);
-	}
-
 	hsub = drm_format_horz_chroma_subsampling(mode_cmd->pixel_format);
 	vsub = drm_format_vert_chroma_subsampling(mode_cmd->pixel_format);
 
-- 
1.7.5.4

